typings/message.d.ts

```diff
@@ -0,0 +1,43 @@
+/// <reference types="node" />
+
+import { CreateTypeDesc } from "./types";
+
+export declare class Section {
+  typecode: number;
+  content: any;
+  multiple: any;
+  constructor(typecode: number, content: any, multiple: any);
+  described(item: any): any;
+}
+
+export declare interface Message {
+  [x: string]: any;
+  toJSON: () => {
+    [x: string]: any;
+  };
+  inspect: () => string;
+  toString: () => string;
+}
+
+export interface message {
+  data_section: (data: any) => Section;
+  sequence_section: (list: any) => Section;
+  data_sections: (data_elements: any) => Section;
+  sequence_sections: (lists: any) => Section;
```

Comment:
the return value for data_section(s), sequence_section(s) should also be opaque. If the internal representation changes, that will not be considered a change in interface.